docs/pages/federating-your-data/generating-tokens.mdx

```diff
@@ -0,0 +1,26 @@
+# Generating Tokens for Accessing Protected Objects
+
+To access a protected object in the Pelican data federation, the client must present a token signed by the Origin where the object is hosted on. The token then is used by Pelican servers to verify the object access permissions granted to the user, alone with other security checks to ensure the token is valid and not tempered by malicious attackers. Pelican takes advantage of JSON Web Token (JWT) to embed permission, user and server identity to the token.
+
+To allow user access the protected objects, the token must be generated by the Pelican Origin. Pelican Origin can be configured to use a third-party OIDC token issuer, in which case the client will be prompted to get a token from the issuer. By default, however, the Pelican Origin needs to generate the token by its built-in issuer and pass it to the client. This page documents different ways to generate the token to access the protected objects.
+
+## Pelican CLI
+
+The Pelican binary comes with a command `pelican origin token create` to generate the token. To generate a valid token, this command **MUST** be run on the same server where the Pelican Origin hosting the target object runs. On the Origin server, run:
+
+```bash copy
+pelican origin token create \
+    --scope "storage.read:/" \
+    --issuer https://localhost:8443 \
+    --claim "wlcg.ver=1.0" \
+    --subject origin \
+    --audience https://wlcg.cern.ch/jwt/v1/any
+```
+
+where:
+
+- `--scope` defines the access permission of the token. It follows [WLCG claim semantics](https://github.com/WLCG-AuthZ-WG/common-jwt-profile/blob/master/profile.md#capability-based-authorization-scope). In this example, the token has `read` permission to the `/` prefix, meaning it can read any objects on this Origin. For other scope options, refer to [WLCG GitHub website](https://github.com/WLCG-AuthZ-WG/common-jwt-profile/blob/master/profile.md#capability-based-authorization-scope).
```

Comment:
Can we document `storage.read`, `storage.write` and `storage.modify` and then also point to the WLCG token scopes docs? I think these are the three a user will want to know without having to link externally. We might also document that fine-grained tokens can be generated by providing an access path, e.g. `storage.read:/path/to/object` will only let you read `/path/to/object`, whereas `storage.read:/` lets you read anything the mapped user is allowed to read.